storage/SystemInfo/CmdParted.cc

```diff
@@ -311,6 +311,11 @@ namespace storage
 
 	entry.region = Region(start_sector, size_sector, region.get_block_size());
 
+	// TODO the parser simply adds the filesystem, name and flags to one
+	// list and then does string matching. This cannot work reliable, just
+	// think of a partition named "esp" containing swap. Use 'parted
+	// --machine'.
```

Comment:
Do we have a card/PBI/bug/whatever for this? Looks like something quite crucial to have addressed before beta phase.